dev-python/python-ldap/python-ldap-3.4.0-r1.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( pypy3 python3_{8..10} )
```

Comment:
Since you're revbumping it, could you also switch to PEP517 build?

Reply:
I'm going to need more output than that.

Reply:
```
ebuild python-ldap-3.4.0-r1.ebuild clean install  
Appending /home/dennis/Development/github_repos/gentoo-portage to PORTDIR_OVERLAY...
 * python-ldap-3.4.0.tar.gz BLAKE2B SHA512 size ;-) ...                                                                                                                                 [ ok ]
>>> Unpacking source...
>>> Unpacking python-ldap-3.4.0.tar.gz to /var/tmp/portage/dev-python/python-ldap-3.4.0-r1/work
>>> Source unpacked in /var/tmp/portage/dev-python/python-ldap-3.4.0-r1/work
>>> Preparing source in /var/tmp/portage/dev-python/python-ldap-3.4.0-r1/work/python-ldap-3.4.0 ...
 * Applying python-ldap-3.4.0-openldap-2.6.patch ...                                                                                                                                    [ ok ]
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/dev-python/python-ldap-3.4.0-r1/work/python-ldap-3.4.0 ...
>>> Source configured.
>>> Compiling source in /var/tmp/portage/dev-python/python-ldap-3.4.0-r1/work/python-ldap-3.4.0 ...
 * python3_9: running distutils-r1_run_phase distutils-r1_python_compile
python3.9 -m pyproject2setuppy build -j 3
Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/lib/python3.9/site-packages/pyproject2setuppy/__main__.py", line 57, in <module>
    main()
  File "/usr/lib/python3.9/site-packages/pyproject2setuppy/__main__.py", line 46, in main
    backend = data['build-system']['build-backend']
KeyError: 'build-system'
 * ERROR: dev-python/python-ldap-3.4.0-r1::gentoo failed (compile phase):
 *   (no error message)
 * 
 * Call stack:
 *     ebuild.sh, line  127:  Called src_compile
 *   environment, line 3192:  Called distutils-r1_src_compile
 *   environment, line 1422:  Called _distutils-r1_run_foreach_impl 'distutils-r1_python_compile'
 *   environment, line  596:  Called python_foreach_impl 'distutils-r1_run_phase' 'distutils-r1_python_compile'
 *   environment, line 2837:  Called multibuild_foreach_variant '_python_multibuild_wrapper' 'distutils-r1_run_phase' 'distutils-r1_python_compile'
 *   environment, line 2373:  Called _multibuild_run '_python_multibuild_wrapper' 'distutils-r1_run_phase' 'distutils-r1_python_compile'
 *   environment, line 2371:  Called _python_multibuild_wrapper 'distutils-r1_run_phase' 'distutils-r1_python_compile'
 *   environment, line  935:  Called distutils-r1_run_phase 'distutils-r1_python_compile'
 *   environment, line 1410:  Called distutils-r1_python_compile
 *   environment, line 1221:  Called esetup.py 'build' '-j' '3'
 *   environment, line 1912:  Called die
 * The specific snippet of code:
 *       "${@}" || die -n;
 * 
 * If you need support, post the output of `emerge --info '=dev-python/python-ldap-3.4.0-r1::gentoo'`,
 * the complete build log and the output of `emerge -pqv '=dev-python/python-ldap-3.4.0-r1::gentoo'`.
 * The complete build log is located at '/var/tmp/portage/dev-python/python-ldap-3.4.0-r1/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/dev-python/python-ldap-3.4.0-r1/temp/environment'.
 * Working directory: '/var/tmp/portage/dev-python/python-ldap-3.4.0-r1/work/python-ldap-3.4.0'
 * S: '/var/tmp/portage/dev-python/python-ldap-3.4.0-r1/work/python-ldap-3.4.0'

```

Reply:
But pep517 mode shouldn't be using pp2sp. I mean `DISTUTILS_USE_PEP517`.